webapp/src/ts/components/filters/freetext-filter/freetext-filter.component.html

```diff
@@ -1,4 +1,4 @@
-<span class="input-group freetext-filter" [class.only-filter]="currentTab === 'contacts'">
+<span class="input-group freetext-filter">
```

Comment:
Does this mean that the `FreetextFilterComponent` no longer needs to get `currentTab` from the store?

Reply:
Nope, we're still using it [here](https://github.com/medic/cht-core/blob/9c1ea5916c926c4b38bab8dd21c0613eba933d50/webapp/src/ts/components/filters/freetext-filter/freetext-filter.component.html#L10)